WooCommerce/Classes/POS/ViewModels/PointOfSaleDashboardViewModel.swift

```diff
@@ -36,13 +36,12 @@ final class PointOfSaleDashboardViewModel: ObservableObject {
     init(itemProvider: POSItemProvider,
          cardPresentPaymentService: CardPresentPaymentFacade,
          orderService: POSOrderServiceProtocol,
-         currencyFormatter: CurrencyFormatter) {
+         currencyFormatter: CurrencyFormatter,
+         totalsViewModel: TotalsViewModel) {
```

Comment:
Since with this change the totalsViewModel is injected and not created, what was the main reason for it? And will we need to change some other parts of the POS code with this?

Reply:
It was a first-cut attempt to improve testability of this class.

This injection was removed in commit 35370c72d7b0f5f967bfaa9304c256416372c12f